pkgs/by-name/sw/sway-audio-idle-inhibit/package.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, meson
+, ninja
+, pkg-config
+, libpulseaudio
+, wayland
+, wayland-protocols
+}:
+stdenv.mkDerivation {
+  pname = "sway-audio-idle-inhibit";
+  version = "0.1.1-unstable-2023-08-09";
+
+  src = fetchFromGitHub {
+    owner = "ErikReider";
+    repo = "SwayAudioIdleInhibit";
+    rev = "c850bc4812216d03e05083c69aa05326a7fab9c7";
+    sha256 = "sha256-MKzyF5xY0uJ/UWewr8VFrK0y7ekvcWpMv/u9CHG14gs=";
+  };
+
+  nativeBuildInputs = [
+    meson ninja pkg-config
+  ];
+
+  buildInputs = [
+    libpulseaudio wayland wayland-protocols
+  ];
+
+  meta = with lib; {
+    description = "Prevents swayidle from sleeping while any application is outputting or receiving audio";
+    homepage = "https://github.com/ErikReider/SwayAudioIdleInhibit";
+    license = licenses.gpl3;
```

Comment:
```suggestion
    license = licenses.gpl3Only;
```

Reply:
Hello, I applied this change as requested. However, I would like to ask, what is the difference between "gpl3" and "glp3Only", and when should I use each? Thanks!

Reply:
Sorry for the missing explanation, but the short answer is gpl3 is deprecated (:

See [https://discourse.nixos.org/t/lib-licenses-gpl3-co-are-now-deprecated/8206](https://discourse.nixos.org/t/lib-licenses-gpl3-co-are-now-deprecated/8206) for more info.

Reply:
I see, the linked post clears everything up! I force pushed both revisions. Thanks for the corrections!